### PR TITLE
Fix lib magic installation

### PR DESCRIPTION
**Pull request by ricardovb**
Fixed bug while installing LibMagic and also fixed bug with missing early log messages on log panel.

### CODE REVIEW

File: setup/setup.bash [hunk 1]
```diff
@@ -307,7 +307,12 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
        fi
    else
        echo "Installing LibMagic with HomeBrew"
-       /usr/local/bin/brew install libmagic
+       OUTPUT="$(/usr/local/bin/brew install libmagic 2>&1 > /dev/null)"
```
Comment (pbalm) [inline setup/setup.bash]: I think this is wrong. You’re sending stderr to stdout with “2>&1” and then you send stdout to dev null. So what’s OUTPUT going to be? The return code? Poor choice of var name then ;-) I think you should remove the part where you send stdout to dev null: “> /dev/null”. Just remove that section of the command.

File: src/main/resources/bash/create_env.bash [hunk 2]
```diff
@@ -52,14 +53,13 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
           echo "LibMagic is required. There are three available options to continue:"
 					echo "=========================================================================="
 					echo "|"
-          echo "|  1 - Relaunch DAVE as root running this command on the terminal:"
-          echo "|      sudo DAVEApp.app/Contents/MacOS/DAVEApp"
+					echo "|  RECOMMENDED: Install LibMagic by yourself running this MacPorts command on the terminal and relanch DAVE:"
+					echo "|      sudo /opt/local/bin/port install file"
```
Comment (pbalm) [inline src/main/resources/bash/create_env.bash]: Port install libmagic?

File: src/main/resources/bash/create_env.bash [hunk 3]
```diff
@@ -52,14 +53,13 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
...
 					echo "|"
-          echo "|  2 - Or install LibMagic by yourself running this MacPorts command on the terminal and relanch DAVE:"
-          echo "|      sudo /opt/local/bin/port install file"
+					echo "|  - Or relaunch DAVE as root running this command on the terminal:"
```
Comment (pbalm) [inline src/main/resources/bash/create_env.bash]: Port install libmagic?

**Review by pbalm — CHANGES_REQUESTED** [review]

**Comment by ricardovb** [comment]
Applied suggested changes, and also improved error checking while installing LibMagic with MacPorts

**Pull request merged**